src/components/Layout/Layout.js

```diff
@@ -0,0 +1,17 @@
+import { Fragment } from 'react';
+
+import Navbar from '../Navbar/Navbar';
+import SideNav from '../Navbar/SideNav';
+
+//Hardcoded SideNav To Layout, Need To Fix With AuthContext
+const Layout = (props) => {
+    return (
+        <Fragment>
+            <Navbar />
+            <SideNav />
+            <main>{props.children}</main>
```

Comment:
Needs a bit more styling, currently it seems like `main` overlaps `SideNav` :))

![image](https://user-images.githubusercontent.com/5387195/149310560-ebfa7f6a-b841-4f76-9658-ad6c5654ace5.png)